app/services/induction/transfer_and_continue_existing_fip.rb

```diff
@@ -25,6 +25,13 @@ def call
                                                preferred_email: email,
                                                mentor_profile:,
                                                school_transfer: true)
+      ## Update old School record and create new one for new school:
+      ## SchoolRecord::Enrol.new(
+      ##  school: school_cohort.school,
+      ##  participant_profile:,
+      ##  joining_date: start_date,
```

Comment:
Maybe we can stop sending start/end dates to induction records and allow them to be set sequentially according to current date/previous end date